### PR TITLE
[SMALLFIX] Add javadoc for IndexedSetConcurrencyTest#TestInfo

### PR DESCRIPTION
**Pull request by saltylin**

### CODE REVIEW

File: tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java [hunk 1]
```diff
@@ -173,24 +173,42 @@ public long runSingleTask() {
...
+     * @param id the id
+     * @param size the size
+     */
     private TestInfo(long id, int size) {
       mId = id;
       mSize = size;
     }
 
+    /**
+     * @return {@link #mId}
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java]: We don't need javadoc for a simple getter like this - the behavior is already clear from the name of the method.

File: tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java [hunk 2]
```diff
@@ -173,24 +173,42 @@ public long runSingleTask() {
...
+     */
     public long getId() {
       return mId;
     }
 
+    /**
+     * @return {@link #mSize}
```
Comment (aaudiber) [inline tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java]: ditto

File: tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java (hunk 1 above)
Comment (saltylin) [inline tests/src/test/java/alluxio/collections/IndexedSetConcurrencyTest.java]: Updated

**Comment by saltylin** [comment]
@aaudiber Thanks for your review and advice. I have removed the unnecessary javadoc.

**Pull request merged**